Nos/Extensions/Font.swift

```diff
@@ -93,7 +72,7 @@ func PlainText(_ localizedStringResource: LocalizedStringResource) -> SwiftUI.Te
     SwiftUI.Text(localizedStringResource)
 }
 
-func Text(_ content: any StringProtocol) -> SwiftUI.Text {
+func BrandText(_ content: any StringProtocol) -> SwiftUI.Text {
```

Comment:
I did a Refactor > Rename on this and would have expected it to properly catch all usages, but I'm not 100% sure it did. It's hard to tell what was using SwiftUI.Text and what was using this `func Text`...